src/ducks/previousAnnotations.js

```diff
@@ -99,6 +99,7 @@ const constructAnnotations = (reductions, frame) => {
         const textOptions = constructText(annotation, i);
         const data = {
           points, frame, textOptions,
+          consensusReached: annotation.consensus_score >= 3,
           previousAnnotation: true,
```

Comment:
Side note: in a future refactor, I'll want to revisit boolean variable names and make them a bit clearer in regards to their true/false nature, e.g. "previousAnnotation" -> "isAPreviousAnnotation". This is a personal note and doesn't affect this PR since `consensusReached` makes enough sense as is. (Unless, dare I suggest... `hasSufficientConsensus` ? `isFullyConsensusified` ?)